antora.yml

```diff
@@ -22,10 +22,13 @@ asciidoc:
     kie_tools_node_min_version: 16.13.2
```

Comment:
We must hold this PR until we have the final version for `kogito_version_redhat`, `quarkus_version`, `vscode_version`, and `kn_cli_version`. @MarianMacik @akumar074

Reply:
cc @ederign

Reply:
@akumar074 can you please update this PR with the following versions?

`kogito_version_redhat: 1.30.0.Final-redhat-00001`
`kn_cli_version : 0.25.0`

Reply:
@ricardozanini Yes, sure.
and will `quarkus_version` and `vscode_version` remain same?

Reply:
Yes! @paulovmr is there any updates in the VSCode Version?

Reply:
@ricardozanini currently the supported VSCode version for the extensions is 1.66.0 or newer.